backend/src/context/pubsub.ts

```diff
@@ -1,19 +1,19 @@
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 import { RedisPubSub } from 'graphql-redis-subscriptions'
 import { PubSub } from 'graphql-subscriptions'
 import Redis from 'ioredis'
 
 import CONFIG from '@config/index'
 
 export default () => {
-  if (!CONFIG.REDIS_DOMAIN || CONFIG.REDIS_PORT || CONFIG.REDIS_PASSWORD) {
+  const { REDIS_DOMAIN, REDIS_PORT, REDIS_PASSWORD } = CONFIG
```

Comment:
The previous if-clause would fail for `REDIS_DOMAIN = 'somedomain'` and `REDIS_PORT = undefined`.